pkg/pillar/containerd/oci.go

```diff
@@ -380,40 +381,10 @@ func (s *ociSpec) updateFromImageConfig(config v1.ImageConfig) error {
 // UpdateMounts adds volume specification mount points to the OCI runtime spec
 func (s *ociSpec) UpdateMounts(disks []types.DiskStatus) error {
 	ociVolumeData := "rootfs"
-	mountDirs := []string{}
 	blkMountPoints := ""
 
-	// We are prepared to deal with the first volume being declared as root,
-	// however any other volume claiming that right would be treated as it
-	// doesn't have MountDir specifies at all
-	id := 0
-	if len(disks) > 0 && disks[0].MountDir == "/" {
```

Comment:
Why did you remove this @deitch ? I think we still need this since `/` is being treated in a very special way (it is not user-assigned)

Reply:
Think about the logic here. It says, "if we have one or more disks, and the first disk has `MountDir == "/"`, then make that the first one.

But that is precisely what we do later. We go through every disk and make its mountDir whatever `disk.MountDir` is.

Reply:
oh yeah -- didn't look at the right place -- sorry about that @deitch